api/src/main/java/ai/djl/util/Utils.java

```diff
@@ -377,4 +382,35 @@ public static Path getNestedModelDir(Path modelDir) {
         }
         return modelDir.toAbsolutePath();
     }
+
+    /**
+     * Gets the value of the specified environment variable.
+     *
+     * @param name the name of the environment variable
+     * @return the string value of the variable, or {@code null} if the variable is not defined in
+     *     the system environment or security manager doesn't allow access to the environment
+     *     variable
+     */
+    public static String getenv(String name) {
```

Comment:
I think it's better have an overload function:
```
public static String getenv(String name, String def) {
```

Reply:
Added